src/BackgroundImage.js

```diff
@@ -1,26 +1,29 @@
 import styled from 'styled-components'
 import PropTypes from 'prop-types'
+import { width, responsiveStyle } from 'styled-system'
 
-const image = props => props.image
-  ? { backgroundImage: `url(${props.image})` }
-  : null
+const image = props =>
+  props.image ? { backgroundImage: `url(${props.image})` } : null
 
-const height = props => props.height
-  ? { height: props.height }
-  : null
+const height = props => (props.height ? { height: props.height } : null)
 
 const BackgroundImage = styled.div`
-  ${image}
-  ${height}
-  background-color: ${props => props.theme.colors.gray};
+  ${image} ${height} ${width} background-color: ${props =>
```

Comment:
I am not sure that I like how it format now.

Reply:
@jxnblk What do you think?

Reply:
I think I'm happy to let the machines decide the best formatting :) We're trying prettier out to stop bikeshedding a little – my guess is that prettier will get better over time and handle some of this embedded CSS syntax a little better in the future

Reply:
@daljeetk In this case try put background-color on first place